py3status/modules/imap.py

```diff
@@ -52,23 +63,16 @@ def check_mail(self):
 
         response = {'cached_until': self.py3.time_in(self.cache_timeout)}
 
-        if not self.new_mail_color:
-            self.new_mail_color = self.py3.COLOR_GOOD
-
-        if mail_count == 'N/A':
-            response['full_text'] = mail_count
-        elif mail_count != 0:
-            response['color'] = self.new_mail_color
-            response['full_text'] = self.py3.safe_format(
-                self.format, {'unseen': mail_count}
-            )
+        if mail_count == self.msg_unavailable:
+            response['color'] = self.py3.COLOR_DEGRADED
```

Comment:
I'd be inclined to do `response['color'] = self.py3.COLOR_ERROR or self.py3.COLOR_BAD` here

`COLOR_ERROR` is part of a long term project around errors and `COLOR_BAD` is the traditional error color